use-cases/payment-automation/using-tokenized-credit-cards.md

```diff
@@ -14,35 +13,34 @@ To see if a user's credit card is already attached to their customer profile, ca
 
 As an integration partner, you will not need to set up your own PCI Proxy account. Rather, when automating payments via the Connector API, you should work with Mews' own account. To do so, call [Get configuration](../../operations/configuration.md#get-configuration) to obtain the `PublicKey` value, found within the [payment card storage](../../operations/configuration.md#payment-card-storage) object. This value will be used as the `merchantId` with PCI Proxy.
 
-### 2. Set up Secure Fields (iframes) to collect cards
-
-Refer to the PCI Proxy documentation on [collecting and storing cards](https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes#3-retrieve-a-transaction-id) and implement steps 1 through 3. See [Examples](https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes#examples) in the documentation for more help.
-1. [Set up Secure Fields](https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes#1-setup-secure-fields).
-2. [Create payment form](https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes#1-setup-secure-fields) to collect card number, CVV, and expiration date*.
-
-*Note on expiration dates:* 
+### 2. Set up Secure Fields (iFrames) to collect cards
 
-Though not required by PCI Proxy, credit card expiration date is listed as a mandatory parameter on the [Add tokenized credit card](../../operations/creditcards.md#add-tokenized-credit-card) endpoint so that it can be displayed within the customer profile in Mews. This can also be used as an additional validation method for the property to know if there is an expired card on file.
+Refer to the [PCI Proxy documentation on Secure Fields (JS)](https://docs.pci-proxy.com/collect/secure-fields-js) for collecting card data through iFrames.
+Implement steps 1 through 3 described in the documentation. See [Examples](https://docs.pci-proxy.com/collect/secure-fields-js#examples) for more help.
 
-To handle this, we recommend that the payment form includes a field for users to input their credit card expiration date, which need not be sent to PCI Proxy, but should be cached, so that the information can be included in the [Add tokenized credit card](../../operations/creditcards.md#add-tokenized-credit-card) API request. 
+1. [Set up Secure Fields](https://docs.pci-proxy.com/collect/secure-fields-js#1.-setup-secure-fields).
+2. [Create the Payment Form](https://docs.pci-proxy.com/collect/secure-fields-js#2.-create-the-payment-form) to collect card number, CVV, and expiration date*.
+3. [Retrieve a Transaction ID](https://docs.pci-proxy.com/collect/secure-fields-js#3.-retrieve-a-transaction-id) using the `merchantId`. Take note of the `transactionId` in the response, which looks like a series of numbers (e.g. `201028230732590232`).
+4. Step 4 \([Obtain the tokens](https://docs.pci-proxy.com/collect/secure-fields-js#4.-obtain-the-tokens)\) is perfomed by the Mews backend, no action is required by the integration partner in this step.
```

Comment:
Maybe have it written like `This step (...) ...` so we don't have the two fours right next to each other, it's not consistent.